### ISSUE TITLE
Optimizations are not enabled by default

### ISSUE THREAD

**Issue by knguy22** [GediminasMasaitis/texel-tuner#9]
This issue was originally found by @dallinson.

He found that adding an `-O3` flag to the `CMakeLists.txt` increased the speed of the tuner by 10x. I tested this change on my own machine and saw similar results.

**Comment by winapiadmin** [comment]
Aren't it supposed to be enabled on Release? (because by default CMake generates as Debug)

**Comment by knguy22** [comment]
Yep that works. A bit embarrassing, but I've learned something new today.
